.github/workflows/blog-notifications.yml

```
@@ -0,0 +1,131 @@
+    - cron: '5 9 * * 1'
+    # Missing blog reminder at 10am UTC
+    - cron: '0 10 * * *'
+  workflow_dispatch: # Allow manual triggering
+
+jobs:
+  daily-rollup:
+    if: github.event.schedule == '0 9 * * *' || github.event_name == 'workflow_dispatch'
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)